src/librustdoc/test/tests.rs

```diff
@@ -277,3 +278,26 @@ test_wrapper! {
     let output = make_test(input, Some("my_crate"), false, &opts, DEFAULT_EDITION);
     assert_eq!(output, (expected, 1));
 }
+
+#[test]
+fn test_html_highlighting() {
```

Comment:
It doesn't make sense to put this test here. Syntax highlighting doesn't have anything to do with running doctests which is what this file is a submodule of. I suggest keeping these tests in `src/librustdoc/html/highlight/tests.rs`.

Reply:
I disagree with you for the location, however I think those tests should be put into a file on their own (like `test/highlight.rs`).